src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -447,16 +448,18 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 "BCP081",
                 $"Resource type \"{resourceTypeReference.FormatName()}\" does not have types available");
 
-            public ErrorDiagnostic SymbolicNameDoesNotExistWithSuggestion(string name, string suggestedName) => new ErrorDiagnostic(
+            public FixableErrorDiagnostic SymbolicNameDoesNotExistWithSuggestion(string name, string suggestedName) => new FixableErrorDiagnostic(
```

Comment:
Since you're adding to `Bicep.Core`, please could you add a few tests to `Bicep.Core.UnitTests` just to assert we're returning the expected data? Doesn't need to be anything super sophisticated, I just want to make sure we're not just depending on the LanguageServer tests to catch regressions.

Reply:
Added two unit tests for the `*WithSuggestions` diagnostics.